README.md

```diff
@@ -5,17 +5,25 @@ Dart package for the Open Food Facts API.
 
 Easily access to more than 1.3 million products from all around the world.
 Open Food Facts is powered by global contributors and is constantly growing thanks to them.
-
+The database in under the OdBL. This means attributing the source and also contributing back any additions, which this packages makes easy to do.
```

Comment:
Not sure I follow this sentence: "This means attributing the source and also contributing back any additions, which this packages makes easy to do."

Also I think you intended to say "makes it easy to do"